README.md

```diff
@@ -1,177 +1,179 @@
-- [lexus_description](https://github.com/tier4/lexus_description.iv.universe)
-
-## How to run
-
-### Supported Simulations
-
-![sim](https://user-images.githubusercontent.com/8327598/79709776-0bd47b00-82fe-11ea-872e-d94ef25bc3bf.png)
-
-### Quick Start
-
-#### Rosbag
-
-1. Download sample map from [here](https://drive.google.com/open?id=1ovrJcFS5CZ2H51D8xVWNtEvj_oiXW-zk).
-
-2. Download sample rosbag from [here](https://drive.google.com/open?id=1BFcNjIBUVKwupPByATYczv2X4qZtdAeD).
-3. Launch Autoware
-
-```sh
-cd AutowareArchitectureProposal
-source install/setup.bash
-roslaunch autoware_launch logging_simulator.launch map_path:=[path] vehicle_model:=lexus sensor_model:=aip_xx1 rosbag:=true
-```
-
-\* Absolute path is required for map_path.
-
-4. Play rosbag
-
-```sh
-rosbag play --clock [rosbag file] -r 0.2
-```
-
-##### Note
-
-- sample map : © 2020 TierIV inc.
-- rosbag : © 2020 TierIV inc.
-  - Image data are removed due to privacy concerns.
-    - Cannot run traffic light recognition
-    - Decreased accuracy of object detection
-
-#### Planning Simulator
-
-1. Download sample map from [here](https://drive.google.com/open?id=197kgRfSomZzaSbRrjWTx614le2qN-oxx).
-
-2. Launch Autoware
-
-```sh
-cd AutowareArchitectureProposal
-source install/setup.bash
-roslaunch autoware_launch planning_simulator.launch map_path:=[path] vehicle_model:=lexus sensor_model:=aip_xx1
-```
-
-\* Absolute path is required for map_path.
-
-3. Set initial pose
-4. Set goal pose
-5. Push engage button.
-   [autoware_web_controller](http://localhost:8085/autoware_web_controller/index.html)
-
-##### Note
-
-- sample map : © 2020 TierIV inc.
-
-#### Running With AutowareAuto
-We are planning propose the architecture and reference implementation to AutowareAuto.
-For the time being, use ros_bridge if you wish to use this repository with AutowareAuto modules.
-You would have to do the message type conversions in order to communicate between AutowareAuto and AutowareArchitectureProposal modules until the architecture is aligned.
-
-For setting up AutowareAuto, please follow the instruction in: https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto
-
-For setting up ros_bridge, please follow the instruction in: https://github.com/ros2/ros1_bridge
-
-#### Tutorial in detail
-
-See [here](./docs/SimulationTutorial.md). for more information.
-
-## References
-
-### Videos
-
-- [Scenario demo](https://youtu.be/kn2bIU_g0oY)
-- [Obstacle avoidance in the same lane](https://youtu.be/s_4fBDixFJc)
-- [Obstacle avoidance by lane change](https://youtu.be/SCIceXW9sqM)
-- [Object recognition](https://youtu.be/uhhMIxe1zxQ)
-- [Auto parking](https://youtu.be/e9R0F0ZJbWE)
-- [360° FOV perception(Camera Lidar Fuison)](https://youtu.be/whzx-2RkVBA)
-- [Robustness of localization](https://youtu.be/ydPxWB2jVnM)
-
-### Credits
-
-- [Neural Network Weight Files](./docs/Credits.md)
+# Autoware (Architecture Proposal)
```

Comment:
@fred-apex-ai what does this file show as being replaced?

Reply:
@esteve Not sure, maybe it's because the file had windows-style line ending and now it's unix? Click the "Rich diff" button to see what actually changed, it's only a few lines